sdk/keyvault/azure-security-keyvault/tests/test_examples_secrets.py

```diff
@@ -0,0 +1,161 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from __future__ import print_function
+import functools
+
+from azure.core.exceptions import ResourceNotFoundError
+from devtools_testutils import ResourceGroupPreparer
+from preparer import VaultClientPreparer
+from test_case import KeyVaultTestCase
+
+
+def print(*args):
+    assert all(arg is not None for arg in args)
+
+
+def test_create_secret_client():
+    vault_url = "vault_url"
+    # pylint:disable=unused-variable
+    # [START create_secret_client]
+
+    from azure.identity import DefaultAzureCredential
+    from azure.security.keyvault.secrets import SecretClient
+
+    # Create a SecretClient using default Azure credentials
+    credentials = DefaultAzureCredential()
+    secret_client = SecretClient(vault_url, credentials)
+
+    # [END create_secret_client]
+
+
+class TestExamplesKeyVault(KeyVaultTestCase):
+    @ResourceGroupPreparer()
+    @VaultClientPreparer(enable_soft_delete=True)
+    def test_example_secret_crud_operations(self, vault_client, **kwargs):
+        secret_client = vault_client.secrets
+
+        # [START set_secret]
+        from dateutil import parser as date_parse
+
+        expires = date_parse.parse("2050-02-02T08:00:00.000Z")
+
+        # create a secret, setting optional arguments
+        secret = secret_client.set_secret("secret-name", "secret-value", enabled=True, expires=expires)
+
+        print(secret.version)
+        print(secret.created)
+        print(secret.enabled)
+        print(secret.expires)
+
+        # [END set_secret]
+        # [START get_secret]
+        secret_version = secret.version
+
+        # get the latest version of a secret
+        secret = secret_client.get_secret("secret-name")
+
+        # alternatively, specify a version
+        secret = secret_client.get_secret("secret-name", secret_version)
+
+        print(secret.id)
+        print(secret.name)
+        print(secret.version)
+        print(secret.vault_url)
+
+        # [END get_secret]
+        # [START update_secret]
+
+        # update attributes of an existing secret
```

Comment:
do we want to mention it updates the latest version of the secret?

Reply:
I dunno. Given that client methods take `version` as the first keyword arg where appropriate (and document it), I lean toward presenting it in fewer places to skirt the prevailing confusion around versioning. Something to think about in doc revisions, I suppose--here I'm just trying to stabilize the live tests.